<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealTimeSetting</class>
 <widget class="QDialog" name="RealTimeSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RealTimeSetting</string>
  </property>
  <widget class="QPushButton" name="pushButton_default">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>默认设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>视频参数</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>视频帧率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fps">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>同一时间最大摘要数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_max_event_num">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>视频保存路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_path">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_path">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="objectpramGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="title">
    <string>目标分析参数</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>视频缩放</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton100">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>以原尺寸对视频进行分析</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton75">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>以原视频尺寸的75%进行分析，加快分析速度</string>
    </property>
    <property name="text">
     <string>75%</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton50">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>以原视频尺寸的50%进行分析，进一步加快分析速度</string>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_width">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_height">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>目标最小面积</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_min_area">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>目标最大面积</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_max_area">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>视频分辨率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分析跳帧</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_jiange">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
